plugins/ci/pkg/ci/trivy.go

```diff
@@ -116,7 +116,7 @@ func (ci *CIScan) GetTrivyReport(manifestImages []trivymodels.Image) (models.Rep
 		return trivyReport, err
 	}
 	// Scan Images with Trivy
-	trivyResults, trivyVersion, err := scanImagesWithTrivy(allImages)
+	trivyResults, trivyVersion, err := scanImagesWithTrivy(ci.config.Options.TempFolder, allImages)
```

Comment:
what is the default when this option is not set?

Reply:
It is `/tmp/_insightsTemp/` - either way, I think Robert already did a similar PR to the folder issue